docs/tutorials/getting_started/command_line.rst

```diff
@@ -38,9 +37,9 @@ Here are some of the plots that are output when running with the latest total nu
 
 The results from the tournament itself (ordered by median score):
 
-.. image:: http://axelrod-python.github.io/tournament/assets/strategies_boxplot.svg
-   :width: 50%
-   :align: center
```

Comment:
Makes sense. I think that this is perhaps a use case for calling an 'outside image' though (?) as the tournament repository is in a way, a separate thing and we want _that_ to be the single source of truth for the results that that image shows...

If you had any thoughts though?